nominatim/tools/country_info.py

```diff
@@ -74,8 +79,21 @@ def setup_country_tables(dsn, sql_dir, ignore_partitions=False):
             lang = props['languages'][0] if len(props['languages']) == 1 else None
             params.append((ccode, partition, lang))
 
+            name = add_prefix_to_keys(props.get('names').get('name'), 'name:')
```

Comment:
Could you make this generic for other prefixes than 'name'. Basically any keys under 'names' should be handled as a prefix.